### PR TITLE
crappy accent

### PR DESCRIPTION
**Pull request by ilinm1**
I have some problems with git again cause Im retarded so Im pulling from fork

### CODE REVIEW

**Comment by ilinm1** [comment]
also, I havent figured out how to use regex lookaround-s so enjoy comas at the sentence start

**Comment by Fogapod** [comment]
Please apply formatting first.
I use pre-commit for formatting and linting.

Commands to use pre-commit:
```sh
$ pip install pre-commit
$ pre-commit install
$ pre-commit run --all-files
```

File: schizophrenia.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+topic_end = ('Ah, sorry, back to the topic...','Uh... Where had we stopped?','Oh sorry, Im talking about random stuff again?','Ah, almost forgot about our talk!')
+
+def switch_topic(m: Match):
+	return " ".join([','+random.choice(topics)]+[random.choice(topic_end)])
+
+def repeat_word(m: Match):
+	n = random.randint(1,2)
+	return " ".join([',' + m.original] * n)
+
+def generate_neologism(m: Match): #pretty crappy function, idk if I should leave it.
```
Comment (Fogapod) [inline schizophrenia.py]: I don't like this comment. This is not the place

File: schizophrenia.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+	neologism = random.choice(start)+random.choice(ending)
+
+	return " ".join([neologism])
+
+class Schizophrenia(Accent):
+	REPLACEMENTS = {
+		r"\w+": {switch_topic: 2, repeat_word: 3, None: 10},
+		f"({'|'.join(re.escape(word) for word in ending)})":generate_neologism()
```
Comment (Fogapod) [inline schizophrenia.py]: I am not sure what is happening on this line. Could you add a comment maybe?

**Review by Fogapod — CHANGES_REQUESTED** [review]

**Comment by Fogapod** [comment]
Let's try this out

**Pull request merged**